_inc/client/state/site/reducer.js

```diff
@@ -105,10 +118,26 @@ export function isFetchingSiteData( state ) {
 	);
 }
 
+/**
+ * Returns true if currently requesting site benefits. Otherwise false.
+ *
+ * @param  {Object}  state Global state tree
+ * @return {Boolean}       Whether benefits are being requested
+ */
 export function isFetchingSiteBenefits( state ) {
 	return !! state.jetpack.siteData.requests.isFetchingSiteBenefits;
 }
 
+/**
+ * Returns true if currently requesting site benefits. Otherwise false.
+ *
+ * @param  {Object}  state Global state tree
+ * @return {Boolean}       Whether benefits are being requested
```

Comment:
update the comment to talk about products.

Reply:
```suggestion
 * @return {Boolean}       Whether products are being requested
```